src/vs/workbench/contrib/chat/browser/chatSlashCommandContentWidget.css

```diff
@@ -4,8 +4,9 @@
  *--------------------------------------------------------------------------------------------*/
 
 .chat-slash-command-content-widget {
-	padding: 0 0.4em;
-	border-radius: 3px;
+	padding: 0.1em 0.3em;
+	top: 7px !important;
```

Comment:
Any way to avoid this with absolute positioning or similar?

Reply:
This component is absolutely positioned, the `!important` exists to override the default value of `top` that is globally set for content widgets here https://github.com/microsoft/vscode/blob/413ad96927c0bb58d625104292c4f8cba38ce55b/src/vs/editor/browser/viewParts/contentWidgets/contentWidgets.ts#L534

Reply:
Actually, this was a bad idea, because this value depends on the input height, so this value works for panel chat and not inline chat, I will remove hardcoding `top`